packages/axes/src/AxisManager.ts

```diff
@@ -109,6 +109,16 @@ export class AxisManager {
     return this._axis[key];
   }
 
+  public setAxis(axis: ObjectInterface<AxisOption>) {
+    Object.keys(axis).forEach((key) => {
+      if (!this._axis[key]) {
+        throw new Error(`Axis ${key} does not exist in Axes instance`);
+      }
+      Object.assign(this._axis[key], axis[key]);
```

Comment:
Same here, you can do it rather like this, and the typescript compiler(+ bundler) will do the work

```js
this._axis[key] = {
  ...this._axis[key],
  ...axis[key]
}
```